src/main/java/net/sf/jabref/gui/help/AboutAction.java

```diff
@@ -33,7 +38,12 @@ public AboutAction(String title, String tooltip, Icon iconFile) {
     @Override
     public void actionPerformed(ActionEvent e) {
         Platform.runLater(() -> {
-            new AboutDialog().show();
+            FXAlert aboutDialog = new FXAlert(Alert.AlertType.INFORMATION, Localization.lang("About JabRef"));
+
+            AboutDialogView view = new AboutDialogView();
```

Comment:
In my opinion it would be slightly cleaner if all this FXAlert stuff is moved to a new `show` method in `AboutDialogView`.